internal/config/config.go

```diff
@@ -0,0 +1,679 @@
+package config
```

Comment:
This file shouldn't be here at all. The `config` package in `pkg/config/` includes these same types and `process` includes the config parser.